zk_monitor/alerts/email.py

```diff
@@ -74,12 +74,16 @@ def _alert(self, path, state, message, params):
         # Create the Alert object. The object takes care of everything from
         # here so we store no reference to it (and let it get garbage
         # collected on its own later)
-        log.debug('Creating Email Alert Message: %s' % message)
-        EmailAlert(
-            subject=subject,
-            body=body,
-            email=to_address,
-            conn=self._mail_backend)
+
+        if type(emails) == str:
+            emails = re.compile('[, ]+').split(emails)
+
+        for e in emails:
+            log.debug('Creating Email Alert Message: %s to %s' % (message, e))
+            EmailAlert(subject=subject,
```

Comment:
Quick question while we're in here.. I'm concerned that if the `EmailAlert()` raises an exception, that `_alert()` method will re-raise the exception all the way up and prevent Tornado from finishing up the other alerters.

Can you dig into the code and see if this is the case? If so, we should patch that up while we're here to make sure that all of the alerts fire off, regardless of whether one fails or not.